score/confirm.go

```diff
@@ -1,24 +1,33 @@
 package score
 
-import "github.com/bitmark-inc/autonomy-api/schema"
+import (
+	"math"
+
+	"github.com/bitmark-inc/autonomy-api/schema"
+)
 
 const (
-	confirmCriteria = 10
+	confirmDataCountThreadHold = 5
```

Comment:
```suggestion
	confirmDataCountThreadHold = 14
```

We use 14-day window for calculation so I suppose the threshold should be 14. If the number if data points is not sufficient, we consider it as 0.

Reply:
remove threadhold